package.json

```diff
@@ -1,18 +1,30 @@
 {
   "name": "lighthouses",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "",
-  "main": "index.js",
+  "main": "index.ts",
   "type": "module",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "type-check": "tsc --noEmit"
   },
-  "author": "",
+  "packageManager": "pnpm@8.13.1",
+  "collaborators": [
+    "Charlie Kornoelje <charlie.kornoelje@tekton.com>",
+    "Duncan Van Keulen <duncan.vankeulen@tekton.com>"
```

Comment:
Thanks for making this change, but let's not put our work emails out there. Could you please remove this commit from history, and then force-push this branch to remove our emails from history?

Let's [link](https://github.com/pmndrs/zustand/blob/517524d2b05843c6a26d2713af08cc0ca1f62f3c/package.json#L195) to our github similar to what they do in this package.

Reply:
Here is a helpful [SO](https://arc.net/l/quote/debomrec) post with how to do this using interactive rebasing.